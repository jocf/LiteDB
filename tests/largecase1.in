set a 1 2 3 4 5
set b 9 9 9 9 9
get a
get b
snapshot
set a 1 2
snapshot
checkout 1
diff a b
inter a b
list snapshots
drop 1
list snapshots
checkout 2
diff a b
cartprod a b
set d 1 4 7 8 3 0 0
cartprod a b d
cartprod d a b
set a 1 2 3 4 5
min a
max a
sum a
len a
BYE
