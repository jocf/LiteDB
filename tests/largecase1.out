> ok

> ok

> [1 2 3 4 5]

> [9 9 9 9 9]

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> [1 2 3 4 5 9]

> []

> 2
1

> ok

> 2

> ok

> [1 2 9]

> [ [1 9] [1 9] [1 9] [1 9] [1 9] [2 9] [2 9] [2 9] [2 9] [2 9] ]

> ok

> [ [1 9 1] [1 9 4] [1 9 7] [1 9 8] [1 9 3] [1 9 0] [1 9 0] [1 9 1] [1 9 4] [1 9 7] [1 9 8] [1 9 3] [1 9 0] [1 9 0] [1 9 1] [1 9 4] [1 9 7] [1 9 8] [1 9 3] [1 9 0] [1 9 0] [1 9 1] [1 9 4] [1 9 7] [1 9 8] [1 9 3] [1 9 0] [1 9 0] [1 9 1] [1 9 4] [1 9 7] [1 9 8] [1 9 3] [1 9 0] [1 9 0] [2 9 1] [2 9 4] [2 9 7] [2 9 8] [2 9 3] [2 9 0] [2 9 0] [2 9 1] [2 9 4] [2 9 7] [2 9 8] [2 9 3] [2 9 0] [2 9 0] [2 9 1] [2 9 4] [2 9 7] [2 9 8] [2 9 3] [2 9 0] [2 9 0] [2 9 1] [2 9 4] [2 9 7] [2 9 8] [2 9 3] [2 9 0] [2 9 0] [2 9 1] [2 9 4] [2 9 7] [2 9 8] [2 9 3] [2 9 0] [2 9 0] ]

> [ [1 1 9] [1 1 9] [1 1 9] [1 1 9] [1 1 9] [1 2 9] [1 2 9] [1 2 9] [1 2 9] [1 2 9] [4 1 9] [4 1 9] [4 1 9] [4 1 9] [4 1 9] [4 2 9] [4 2 9] [4 2 9] [4 2 9] [4 2 9] [7 1 9] [7 1 9] [7 1 9] [7 1 9] [7 1 9] [7 2 9] [7 2 9] [7 2 9] [7 2 9] [7 2 9] [8 1 9] [8 1 9] [8 1 9] [8 1 9] [8 1 9] [8 2 9] [8 2 9] [8 2 9] [8 2 9] [8 2 9] [3 1 9] [3 1 9] [3 1 9] [3 1 9] [3 1 9] [3 2 9] [3 2 9] [3 2 9] [3 2 9] [3 2 9] [0 1 9] [0 1 9] [0 1 9] [0 1 9] [0 1 9] [0 2 9] [0 2 9] [0 2 9] [0 2 9] [0 2 9] [0 1 9] [0 1 9] [0 1 9] [0 1 9] [0 1 9] [0 2 9] [0 2 9] [0 2 9] [0 2 9] [0 2 9] ]

> ok

> 1

> 5

> 15

> 5

> bye
